plugins/module_utils/crypto/_obj2txt.py

```diff
@@ -1,7 +1,15 @@
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is licensed under the
+# Apache 2.0 License. Modules you write using this snippet, which is embedded
+# dynamically by Ansible, still belong to the author of the module, and may assign
+# their own license to the complete work.
+
 # This excerpt is dual licensed under the terms of the Apache License, Version
```

Comment:
For example, the proper SPDX license identifier for this file is `# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause`.